### PR TITLE
feat: used only missing items amount

### PR DESCRIPTION
**Pull request by raphael-arce**
used only missing items amount instead of shoppping list length to display the number on the toggle shopping list button

### CODE REVIEW

File: src/utils/timeStampAndSortBy.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ export const toDate = (timeStamp) => {
   if (minutesToNow < 10) {
     return 'a few minutes ago';
   }
-  
+
```
Comment (raphael-arce) [inline src/utils/timeStampAndSortBy.js:42]: this was fixed by linting. I'm wondering why this was not changed by the linting before or noticed in the github action checking whether linting is ok 🤔

File: src/store/lists/lists.js [hunk 2]
```diff
@@ -91,6 +91,10 @@ export const useListsStore = create((set, get) => ({
     set(() => ({ shoppingList: new Map() }));
   },
 
+  getMissingItemsAmount: () => {
+    return [...get().shoppingList.entries()].filter(([, value]) => !value.isBought).length;
```
Comment (raphael-arce) [inline src/store/lists/lists.js:95]: Here I wasn't sure if I should add more steps to make it more readable. for Example:

```javascript
getMissingItemsAmount: () => {
  const entries = [...get().shoppingList.entries()];

  const missingItems = entries.filter(([, value]) => !value.isBought);

  return missingItems.length;
}
```

File: src/utils/timeStampAndSortBy.js (hunk 1 above)
Comment (samiletuns) [reply src/utils/timeStampAndSortBy.js:42]: That's weird indeed 🧐

File: src/store/lists/lists.js (hunk 2 above)
Comment (samiletuns) [reply src/store/lists/lists.js:95]: I'd it's still quite readable and wouldn't require more const like in the example!

**Pull request merged**